lib/rest/Twilio.js

```diff
@@ -137,6 +137,11 @@ function Twilio(username, password, opts) {
   }
 
   if (!this.accountSid.startsWith('AC')) {
+
+    if (this.accountSid.startsWith('SK')) {
```

Comment:
Thank you for the PR @ducsuus!

Could you please add a test to verify this functionality?

We will be making this change internally, since this file is generated.